### ISSUE TITLE
2.0.13.2 hotfix broke ActiveRecord::refresh() method

### ISSUE THREAD

**Issue by PowerGamer1** [yiisoft/yii2#15933]
### What steps will reproduce the problem?
```PHP
$model = User::findOne(['id' => 123]); // user 123 exists in database
$model->refresh();
```
### What is the expected result?
No errors.

### What do you get instead?
yii\base\InvalidParamException
Key "user.id" is not a column name and can not be used as a filter

### Additional info
The example code above works fine in Yii2 2.0.13.1.

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.13.2

**Comment by samdark** [comment]
I think it's 2.0.15, not 2.0.13.2.

**Comment by PowerGamer1** [comment]
The example above works fine in 2.0.15, the affected version that was broken is 2.0.13.2.

**Comment by samdark** [comment]
Ah... OK.

**Comment by cebe** [comment]
similar to #15931, going to fix it today.

**Comment by cebe** [comment]
here is the patch to apply to 2.0.13. https://github.com/yiisoft/yii2/commit/c70308a92adbb1d36eb16f4d6bb3409a01fa80c8 please comment on the commit, there is no pull request.

**Comment by cebe** [comment]
here is the patch to apply to 2.0.12. 6eb7320

**Comment by cebe** [comment]
fixed in 2.0.13.3 and 2.0.12.2. Thanks for reporting!

**Comment by cebe** [comment]
Same fix is applied for  #15931 in 2.0.15.1
